soc/silabs/silabs_siwx91x/siwg917/nwp.c

```diff
@@ -109,6 +109,11 @@ int siwg91x_get_nwp_config(int wifi_oper_mode, sl_wifi_device_configuration_t *g
 		if (IS_ENABLED(CONFIG_BT_SILABS_SIWX91X)) {
 			LOG_WRN("Bluetooth is not supported in AP mode");
 		}
+
+		if (IS_ENABLED(WIFI_SILABS_SIWX91X_LIMIT_PACKET_BUF_PER_STA)) {
```

Comment:
Does it works? I believe you mean `CONFIG_WIFI_SILABS_SIWX91X_LIMIT_PACKET_BUF_PER_STA`.

Reply:
Not fixed.